genesis/genesis.json

```diff
@@ -0,0 +1,191 @@
+{
+  "genesis_time": "2021-11-22T14:59:05.473199025Z",
+  "chain_id": "devnet-1",
+  "initial_height": "1",
+  "consensus_params": {
```

Comment:
I think this is wrong. I don't understand why we would need to create a genesis file separately per gentx submission.